V93 koecher candi (#62)
dealii/release-papers: 9.3/paper.tex

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Source based toolchain installer}
\label{subsec:candi}

The requirement to download, compile and install \dealii{} and its dependencies from

> source is of mayor interest for many \dealii{}-users.

Reviewer: ```suggestion
source is of major interest for many \dealii{}-users.
```

%

> This can be of importance on arbitrary Linux-based operating systems for which

Reviewer: ```suggestion
Compiling dependencies from source can be of importance on arbitrary Linux-based operating systems for which
```

> binary-install package aren't available or on compute clusters and other machines

Reviewer: ```suggestion
binary packages aren't available or on compute clusters and other machines
```

without root privileges for the user to install system dependencies.

> The source based installation of \dealii{} or complex toolchains of multiple libraries

Reviewer: ```suggestion
The source based installation of \dealii{} and it's many dependent libraries
```

Reviewer 2: It should be "its", not "it's". I'll fix that myself.

> working together with \dealii{} can be done with the \candi{} script tool for various

Reviewer: ```suggestion
can be done with the \candi{} script tool for various
```

Linux operating systems, within the Windows Subsystem Linux (WSL) and on OSX (experimental).
%
The general assumption is that a C, C++ and Fortran compiler and suitable MPI-compilers
for the base compilers as well as the corresponding development system packages are
available.

\candi is a bash-script based tool, it is the abbreviation of ``compile and install''
and it is released under the GNU LGPL v3.0.
%
The origin of \candi{} is a fork made in 2013 from \texttt{dorsal},
a retired source based installer for the \texttt{FEniCS} library.
%
The \candi{} tool can be found via the download page of \dealii{} since version 8.5

> and is under active maintenance.

Reviewer: ```suggestion
or from github.com/dealii/candi and is under active development.
```
